day-1/slides.md

```diff
@@ -0,0 +1,589 @@
+# N8N Tutorial - Day 1
+
+## Pengenalan N8N untuk Automation
+
+---
+
+<!-- SECTION 1: Introduction -->
+# Introduction N8N
+
+---
+
+## Apa itu N8N?
+
+![N8N Logo](day-1/images/n8n-logo.png)
+<!-- Placeholder: Logo N8N -->
+
+- **N8N** adalah platform automation workflow yang open-source<br>
+- Memungkinkan integrasi antar aplikasi tanpa coding<br>
+- Visual workflow builder<br>
+- Self-hosted solution
+
+Note:
+N8N memungkinkan kita untuk mengotomatisasi tugas-tugas repetitif dengan cara yang visual dan mudah dipahami.
+
+---
+
+## Mengapa N8N?
+
+- 🔧 **No-code/Low-code** approach<br>
+- 🔗 **300+ integrations** tersedia<br>
+- 🏠 **Self-hosted** - kontrol penuh atas data<br>
+- 💰 **Open source** - gratis untuk digunakan<br>
+- 🎨 **Visual interface** - mudah dipahami
+
+![Use Cases](day-1/images/use-cases.png)
+<!-- Placeholder: Diagram use cases N8N -->
+
+---
+
+## Target Hari Ini
+
+Setelah sesi ini, peserta dapat:
+
+- ✅ Memahami interface N8N<br>
+
+- ✅ Membuat workflow sederhana<br>
+
+- ✅ Menggunakan berbagai trigger
+
+- ✅ Mengelola kredensial
+
+- ✅ Memonitor eksekusi workflow
+
+---
+
+<!-- SECTION 2: User Interface -->
+# Berkenalan dengan User Interface
```

Comment:
Coba bikin agar rather than berkenalan dengan user interface conceptnya nyicip bagaimana the very basic workflow, nanti jelas akan dijelasin lebih in depth, jadi nanti dari pengajar akan mengajarkan ini loh sample app, credentialnya sudah disetup sama pengajar, jadi user langsung kebayang dulu sebelum nanti in depth